aimsprop/fms90.py

```diff
@@ -334,11 +334,11 @@ def _create_frames_saddle(
 
 
 def parse_fms90(
-    filepath: str,
-    scheme: str = "mulliken",
-    cutoff_time: float = None,
-    cutoff_saddle: float = 1.0e-4,
-    initial_I: int = None,
+        filepath: Path,
```

Comment:
This was left as a string to not break external callers who use this method. Are we sure we want to change this from `str` to `Path`? I'm not sure what other parts of code use this function and if they are passing a `str` instead of a `Path` object. I'm in favor of enforcing the use of `Path` objects but want to make sure this is compatible with established uses. Is there anywhere in the code that we utilize `parse_fms90` and we pass a `str` for filepath?

As a note, this is the sort of thing that would be considered a "breaking change" to our library. This means that anyone who has written scripts that use the `parse_fms90` function would now have broken scripts because they likely passed a `str` before, and now we expect a `Path` object. One should make changes like this to public methods very judiciously as they render the entire package broken for all backwards compatible use!

Given that we don't have lots of external users, this is probably a good idea, but want to make you aware of the implications of changing public method signatures.

Reply:
yes, alessio and i spoke about this issue this morning. we checked the rest of the library and it doesnt call this function. Since there are only a couple of aimsprop users at the moment it is probably a good idea but i am happy to revert back to string if better.